### PR TITLE
Fixes in trakt_emit: unused imports, UnboundLocalError of var `eps` #3033

### PR DESCRIPTION
**Pull request by antivirtel**
Responsible for testing is qvazzler.

### CODE REVIEW

**Comment by antivirtel** [comment]
Maybe an `eps = null` would be better, since rarely there used to be 0th episodes, pilots, etc...

**Comment by qvazzler** [comment]
Hi Gaz, please merge so I can use trakt_emit again. :+1:

File: flexget/plugins/input/trakt_emit.py [hunk 1]
```diff
@@ -103,6 +101,8 @@ def on_task_input(self, task, config):
                             else:
                                 epn += 1
                         break
+                    else:
+                        eps = 0;
```
Comment (stevezau) [inline flexget/plugins/input/trakt_emit.py]: not sure why this line ends in a semicolon?

Also you could cleanup the above logic 

IE:
```
if config['position'] == 'next':
                             if seas['completed'] >= seas['aired']:
```

to:
```
if config['position'] == 'next' and seas['completed'] >= seas['aired']:
```

**Comment by antivirtel** [comment]
The semicolon is just a routine for me @stevezau, mainly, I'm Java developer. I'll remove it in the next commit, but I'm still waiting for @gazpachoking's help, to figure out what to do with that `eps = 0`.

**Comment by gazpachoking** [comment]
I think probably something like this would be okay as an `else` to the for loop.
```
                else:
                    if config['position'] == 'next':
                        eps = epn = 1
                    else:
                        # There were no watched/collected episodes, nothing to emit in 'last' mode
                        continue
```

**Comment by gazpachoking** [comment]
Hmm, I meant the else block on the for loop, rather than that if statement. @cvium Did you want to give this a shot so @antivirtel isn't doing it blind?

**Comment by cvium** [comment]
Sure... once it's been updated.

**Comment by antivirtel** [comment]
@gazpachoking did you want to place that code block into this `for` loop's 1st or 2nd `if`?( https://github.com/Flexget/Flexget/blob/develop/flexget/plugins/input/trakt_emit.py#L92 or https://github.com/Flexget/Flexget/blob/develop/flexget/plugins/input/trakt_emit.py#L97 )

**Comment by cvium** [comment]
@gazpachoking is suggesting a `for-else` loop. It should not be inside the `for`-loop.

**Comment by antivirtel** [comment]
Ah I hope I get it now (https://docs.python.org/2/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops) - so an `else` into this `for`: https://github.com/Flexget/Flexget/blob/develop/flexget/plugins/input/trakt_emit.py#L90 !

**Comment by cvium** [comment]
Yes.

**Comment by cvium** [comment]
I am unable to test this for the next 2 weeks, so if someone else could handle this it'd be cool.

**Comment by qvazzler** [comment]
Really crammed up at work around this time guys.. Will try when I get the chance.

**Comment by qvazzler** [comment]
Just tested your PR again Antivirtel, it's working! :+1: :+1: :+1:

**Comment by gazpachoking** [comment]
Thanks everyone!

**Pull request merged**